src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.html

```diff
@@ -1,45 +1,57 @@
 <kbn-management-app section="kibana">
   <kbn-management-indices>
-    <div ng-controller="managementIndicesCreate" class="kbn-management-indices-create">
```

Comment:
The only definition I see of this style is in `src/core_plugins/kibana/public/management/styles/main.less`. Nothing else references this class, so you should probably remove it from this less file.